### DISCUSSION TITLE
Launching a Statiq web project via Visual Studio Code

### DISCUSSION THREAD

**Discussion by blachawk** [statiqdev/Discussions#29] [Q&A]
I have found great success in launching my first statiq web project (as well as with the Clean Blog package) all from Visual Studio.  But when I attempt to create & run this project form VSCode, it doesn't feel/load the same.  My question...

* Is this project suitable to compile fully in VSCode? If so what extensions or settings do I need in place to make it work 100% from this environment?

**Comment by daveaglick** [comment]
It's all stock .NET (there's nothing exotic or fancy going on build-wise), so it should work as well as any other .NET project as far as I know. I actually use VS Code quite a bit, but generally run it from the integrated terminal using `dotnet run -- preview`. Are you attempting to compile/run directly from the "play" button in VS Code? If so, do you have a `.vscode/tasks.json` file and what does it contain?

**Reply by blachawk** [reply]
You may have sparked something.  Am I suppose to have processes setup in that task.json file?  I have not paid attention to a .vscode folder process for this project.  Let me see and get back with you...
